main/src/vtr_lidar/src/modules/localization/localization_icp_module.cpp

```diff
@@ -329,6 +329,7 @@ void LocalizationICPModule::run_(QueryCache &qdata0, OutputCache &,
     *qdata.T_r_v_loc = T_r_v_icp;
     // set success
     *qdata.loc_success = true;
+    CLOG(WARNING, "lidar.localization_icp") << "The resulting transform is:" << T_r_v_icp;
```

Comment:
This feels like an INFO, but might be part of a larger required inspection of log messages repo wide.